<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Campo electrico</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbDistribution">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>181</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Distribuciones</string>
    </property>
    <widget class="QRadioButton" name="rbLine">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Línea finita</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbRing">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Anillo</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbDisk">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Disco</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbParameters">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Parámetros</string>
    </property>
    <widget class="QLabel" name="lblDimension">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dimension (m)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDimension">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPointP">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Punto P (m)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCalculate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPointP_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.000001</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPointP_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Carga (C)</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblValue">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Value: 10</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbResults">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Resultados</string>
    </property>
    <widget class="QLabel" name="lblField">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Campo eléctrico (N/C)</string>
     </property>
    </widget>
    <widget class="QOpenGLWidget" name="openGLWidget">
     <property name="geometry">
      <rect>
       <x>-480</x>
       <y>350</y>
       <width>300</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtField">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
